src/System.AppContext/src/System/AppContext.cs

```diff
@@ -44,6 +45,19 @@ public static object GetData(string name)
             return null;
         }
 
+        public static event UnhandledExceptionEventHandler UnhandledException
```

Comment:
I don't believe we will need to but it really depends on how we implement some of the stuff like UnhandledException. I'm good with moving AppContext out of S.P.CoreLib even on CoreCLR if that works, I've not looked at it in detail to see if there are any issues with that. If we do move it out then we can share more of the code in corefx and we would then need some "Augments" type in CoreLib for things like this.